.github/workflows/test.yml

```
@@ -1,139 +1,186 @@
+          poetry run pytest tests/unit/ \
+            --ignore=tests/unit/lc/ \
+            --cov=fmp_data \
+            --cov-exclude=fmp_data/lc/ \
+            --cov-report=xml:coverage-core.xml \
+            --cov-report=term-missing \
+            -v
+
+      - name: Upload core coverage
+        uses: codecov/codecov-action@v5
+        with:
+          file: ./coverage-core.xml
```

actionlint (1.7.7)

input "file" is not defined in action "codecov/codecov-action@v5". available inputs are "binary", "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "files", "flags", "gcov_args", "gcov_executable", "gcov_ignore", "gcov_include", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugins", "report_code", "report_type", "root_dir", "skip_validation", "slug", "swift_project", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)